### ISSUE TITLE
Benchmark Existing Models

### ISSUE THREAD

**Issue by Demirrr** [dice-group/Ontolearn#273]
1. Create few learning problems 
2. Use learners **with default parameters**
3. Report Results in README.md

By this, we can
1.  Show the trade-off between models numericaly to the users
2.  Ensure that suitable default parameters are selected.

**Comment by Demirrr** [comment]
Can you @alkidbaci evaluate existing models on the family benchmark dataset with these learning problems (https://github.com/dice-group/Ontolearn/blob/develop/examples/synthetic_problems.json)?

Ideally, we should create a table, 

```
LP                         CELOE              Evolearner

Grandgrandfather     0.8, 0.7, 13.2         0.9, 0.6, 10.2
```
where numbers correspond to F1 score, Accuracy, and Runtimes.

**Comment by Demirrr** [comment]
The initial version will be pushed soon to development branch

**Comment by Demirrr** [comment]
Thank you! Solved with #296
